packages/phones/src/phone-service/PhoneService.spec.ts

```diff
@@ -51,11 +51,11 @@ test.each([
   ['+15062345678', 'CA', '506-234-5678'],
   ['+150623456789', 'US', '50623456789'],
   ['+1506234567890', 'US', '506234567890'],
-  ['+15062345678,+15062345679,+15062345680', 'CA', '506-234-5678'],
+  ['+15062345678,+15062345679,+15062345680', 'US', '5062345678'],
```

Comment:
@umrzoq-toshkentov What the reason of this change?

Reply:
because the latest version of `awesome-phonenumber`, there is some difference on input and national. That's why I have updated some specs there. I think it does not break anything.

Reply:
because on the latest change it is changed a bit of formatting. That's why I just updated some specs here.
There is difference with input.

`Before`
![image](https://github.com/superdispatch/web-ui/assets/118124821/2ade3cb7-7f26-45ea-b5e7-b79a6049947f)

________________________________________________

`The Latest`

![image](https://github.com/superdispatch/web-ui/assets/118124821/ef2822dc-b709-4876-82eb-e735539bf942)

Reply:
@umrzoq-toshkentov In you example, difference in `input`. But in this PR `nationalNumber` is changed. Can you clarify, why?

Reply:
@timurhamitov Because on formating we have a function `getInfo` function and used `input` value to generate nationalNumber. So it affects now 
<img width="823" alt="image" src="https://github.com/superdispatch/web-ui/assets/118124821/7599b778-f27f-42e3-b357-a950b3b8977f">

Reply:
@timurhamitov I think that's why there is removed `-` on the latest version